### ISSUE TITLE
Broken group info parsing in system package

### ISSUE THREAD

**Issue by gongled** [essentialkaos/ek#148]
### Bug report

**System info:** CentOS 6 x84_64

* **Version used:** ek.v9
* **OS (e.g. from `/etc/os-release`):** CentOS release 6.8
* **Kernel (e.g. `uname -a`):** Linux 2.6.32-642.15.1.el6.x86_64 #1 SMP Fri Feb 24 14:31:22 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
* **Go version (e.g. `go version`):** 1.8.1

**Steps to reproduce:**

1. Build binary from the following source code:

```go
package main

import (
	"fmt"
        "pkg.re/essentialkaos/ek.v9/system"
)

func main() {
	user, err := system.CurrentUser()

	if err != nil {
		fmt.Println(err.Error())
	} else {
        fmt.Println(user.UID, user.GID, user.Name)
		for _, group := range(user.Groups) {
                        fmt.Println(group.Name)
		}
	}
}
```

2. Run binary on target system with LDAP installed and configured. Here is the output of `id` command:
```
uid=10591(ggoncharov) gid=10591(ggoncharov) groups=10591(ggoncharov),10507(admins),10510
```

**Expected behavior:**

```
10591 10591 ggoncharov
ggoncharov
admins
```

**Actual behavior:**

```
panic: runtime error: index out of range

goroutine 1 [running]:
pkg.re/essentialkaos/ek.v9/system.parseGroupInfo(0xc4200b0058, 0x5, 0x1)
	/golang/src/pkg.re/essentialkaos/ek.v9/system/user.go:347 +0x14f
pkg.re/essentialkaos/ek.v9/system.appendGroupInfo(0xc42008e120)
	/golang/src/pkg.re/essentialkaos/ek.v9/system/user.go:261 +0x1a9
pkg.re/essentialkaos/ek.v9/system.LookupUser(0xc42000e540, 0xb, 0x0, 0x1, 0x549840)
	/golang/src/pkg.re/essentialkaos/ek.v9/system/user.go:155 +0x7d
pkg.re/essentialkaos/ek.v9/system.CurrentUser(0x0, 0x0, 0x0, 0xc420037f10, 0x0, 0xc420037f78)
	/golang/src/pkg.re/essentialkaos/ek.v9/system/user.go:128 +0x57
main.main()
	/golang/src/github.com/test/main.go:9 +0x4e
```

@andyone I suppose we should ignore groups without names. Sometimes group identifier may exist without text description what name of the group it is.

**Comment by andyone** [comment]
Hi! Thanks for your report.

> I suppose we should ignore groups without names.

I'm not sure that's right, because we lose some information in this case. I think we must search info using `LookupGroup` method if group description is empty.

**Comment by andyone** [comment]
I will fix it later today.

**Comment by gongled** [comment]
@andyone but what should we do in case group identifier have multiple matches?

**Comment by andyone** [comment]
Do you mean if a user has two groups with same GID? I think `getent` will return first matched group.

**Comment by andyone** [comment]
Fixed in #147
